lib/msal-common/src/unifiedCache/serialize/JsonKeys.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { CacheHelper } from "../utils/CacheHelper";
+
+/**
+ * Map for AccessTokenCache entity serialization and deserialization
+ */
+export class AccessTokenCacheMaps {
+    static toCacheMap = {
+        homeAccountId: "home_account_id",
+        environment: "environment",
+        credentialType: "credential_type",
+        clientId: "client_id",
+        secret: "secret",
+        realm: "realm",
+        target: "target",
+        cachedAt: "cached_at",
+        expiresOn: "expires_on",
+        extendedExpiresOn: "extended_expires_on",
+        refreshOn: "refresh_on",
+        keyId: "key_id",
+        tokenType: "token_type"
+    };
+
+    static fromCacheMap = CacheHelper.swap(AccessTokenCacheMaps.toCacheMap);
+}
+
+/**
+ * Map for IdTokenCache entity serialization and deserialization
+ */
+export class IdTokenCacheMaps {
+    static toCacheMap = {
+        homeAccountId: "home_account_id",
+        environment: "environment",
+        credentialType: "credential_type",
+        clientId: "client_id",
+        secret: "secret",
+        realm: "realm"
+    };
+
+    static fromCacheMap = CacheHelper.swap(IdTokenCacheMaps.toCacheMap);
+}
+
+/**
+ * Map for RefreshTokenCache entity serialization and deserialization
+ */
+export class RefreshTokenCacheMaps {
+    static toCacheMap = {
```

Comment:
not sure I like having static variables. Should make these constants.

Reply:
Cannot call functions, in this space specifically `swap` cannot be called unless it is `static`. Ideas welcome for a more efficient way to achieve this mapping.